src/DateSelector/index.js

```diff
@@ -370,6 +372,10 @@ DateSelector.propTypes = {
    * Triggered when a preset is selected.
    */
   onPresetChange: func,
+  /**
+   * @see [Popover](#popover) - The placement to be used on Popover component
+   */
+  placement: string,
```

Comment:
Here we can use `oneOf` because we provide only 2 positions for the `placement`.
```suggestion
  placement: oneOf(['bottomStart', 'bottomEnd']),
```